control_modes:
  idle: false

  teleop:
    surge:      {p: 1.0,  i: 3.0,   d: 5.0,  i_max: 10, i_min: -10}
    yaw_rate:   {p: 2.0,  i: 0.5,   d: 3.0,  i_max: 20, i_min: -20}
    pitch_rate: {p: 2.0,  i: 0.5,   d: 3.0,  i_max: 20, i_min: -20}

  flight:
    z:        {p: 20.0,    i: 2.0 ,  d: 5.0,  i_max: 30, i_min: -30}
    surge:    {p: 50.0,    i: 10.0,   d: 0.0,   i_max: 30, i_min: -30}
    pitch:    {p: 10.0,    i: 0.0,   d: 10.0,  i_max: 20, i_min: -20}
    yaw:      {p: 10.0,    i: 1.0,   d: 20.0,  i_max: 30, i_min: -30}

  hold_dof:
    surge:    {p: 50.0,    i: 1.0,   d: 2.0,   i_max: 20, i_min: -20}
    z:        {p: 20.0,   i: 1.0 ,  d: 5.0,  i_max: 20, i_min: -20}
    pitch:    {p: 10.0,   i: 0.01,  d: 5.0,  i_max: 20, i_min: -20}
    yaw:      {p: 10.0,   i: 0.2,   d: 5.0,  i_max: 20, i_min: -20}